### DISCUSSION TITLE
Text Objects

### DISCUSSION THREAD

**Discussion by Pheon-Dev** [folke/flash.nvim#88] [Ideas]
Just a spin off from the capabilities of `flash` using `R`, `r`, `S`, say an improvement on treesitter's current implementation of text objects which is currently chaos for me that I have about 5 plugins (`nvim-treesitter/nvim-treesitter-textobjects`, `chrisgrieser/nvim-various-textobjs`, `glts/vim-textobj-comment`, `kana/vim-textobj-user` plus my own for quotes [\`,  ' and "]) to make it work right, which makes it kinda convoluted. Hell, I'll be glad to contribute to `folke/text-objects.nvim` if it will ever be. Thanks.

**Comment by folke** [comment]
This might be of interest: https://github.com/folke/flash.nvim/discussions/45
Basically custom treesitter text objects with flash

**Reply by Pheon-Dev** [reply]
Cool, I'll check it out.

**Comment by folke** [comment]
Also, check `mini.ai` which is what I'm using.

It also supports treesitter objects. Check the LazyVim config for mini.ai to see how to properly set it up

**Reply by Pheon-Dev** [reply]
I sure will try it again, hopefully there're some features I had missed in the previous attempts that I tried using it. Thanks.
